news/119.feature

```diff
@@ -0,0 +1,17 @@
+- added `wsgi-ini-template` option for specifying a custom template for the
+  generation of the `wsgi.ini` configuration file
+  [ajung]
+
+- advanced WSGI logging configuration options for specifying arbitrary logging
+  handlers for event and access log:
+
+  - access-log-handler 
+  - access-log-args 
+  - access-log-kwargs 
+  - event-log-handler 
+  - event-log-args
+  - event-log-kwargs
+  [ajung]
+
+- added documentation and examples for advanced WSGI logger configuration
+  [ajung]
```

Comment:
in a news file, all must be in one line (yeah, annoying, I know). To split in multiple entries just create multiple files, like 119-1.feature, 119-2.doc, ...

Reply:
@zopyx could you sort this out?